Hazel/src/Hazel/Core/Base.h

```diff
@@ -87,5 +47,4 @@ namespace Hazel {
 	{
 		return std::make_shared<T>(std::forward<Args>(args)...);
 	}
-
 }
```

Comment:
Similarly at the closing bracket of the namespace we have an empty line
```suggestion
	}

}
```